workflow_array_ephys/paths.py

```diff
@@ -4,12 +4,9 @@
 
 def get_ephys_root_data_dir():
     root_data_dirs = dj.config.get('custom', {}).get('ephys_root_data_dir', None)
-
-    return root_data_dirs
-
+    return root_data_dirs if root_data_dirs else None
```

Comment:
```suggestion
    return root_data_dirs or None
```

Reply:
Resolved this function to the following during the rebase process
`return dj.config.get('custom', {}).get('ephys_root_data_dir', None)`